code-challenges.js

```diff
@@ -31,38 +67,92 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const yourDegrees = (temp) => {
+    if (temp < 212) {
+        return `${temp} is below boiling point`
+    } else if (temp === 212) {
+        return `${temp} is at boiling point`
+    }
+    else if (temp > 212) {
+        return `${temp} is above boiling point`
+    }
+}
+console.log(yourDegrees(temp1))
+console.log(yourDegrees(temp2))
+console.log(yourDegrees(temp3))
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+// Dodgers are better...
+// Divide this into two parts: 1. combine the arrays 2.return the length
+// To combine the two arrays and preserve both of their values, use the ".concat" accessor. 
+    //Create a new variable "concatArrays" to describe the data being combined of the two variables provded.
+    // Set the new variable to padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// Next, to determine the length of the new variable, use the ".push" mutator at the end of the new variable. 
+    // This will return 9 becuase nine numbers exist in the new array, since they were combined.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+var concatArrays =padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// console.log(concatArrays)
+console.log(concatArrays.push())
+ 
+// padres1984WorldSeriesRuns.push(padres1998WorldSeriesRuns)
+// console.log(padres1984WorldSeriesRuns.push(padres1998WorldSeriesRuns)) COME BACK TO THIS!!
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
+// First, create a new variable to set at the value of the old variable (currentCohort)
+    // use the built-in accessor .split() to split the string into an array of characters
+    // This should return 'G' 'o' 'l' 'f' ' ' '2' '0' '2' '2'
+// Next reverse the order using the .reverse() mutator 
+    //This should return  '2', '2', '0','2', ' ', 'f','l', 'o', 'G'
+//Lastly, to join it alltogether to acheive the expected output of "2022 floG" use the .joing() accessor 
+// Holy cow, this took an embarrassingly long time to figure out the answer. I knew the accessors and mutators I had to input, but I am having trouble understanding why I have to console.log(newCohort) each time, when terminal returns the answer I am looking for if I didn't type console.log. For example, let newCohort = currentCohort.split("") gives me the array in characters already in the terminal, so I keep getting stuck on why I have to re-call console.log. I dont know if this made any sense, but I would love to describe this over zoom. 
```

Comment:
I think I am tracking on what you mean here.  Each built in method is in a sense building onto each other.  To view that process, you would want to console log, but ultimately the intent is to get the expected output at the end of this.  You can also stack the methods together for an all in one approach: currentCohort.split("").reverse().join("")